lib/mail_room/mailbox_watcher.rb

```diff
@@ -57,8 +57,14 @@ def quit
     end
 
     private
+
     def connection
-      @connection ||= ::MailRoom::IMAP::Connection.new(@mailbox)
+      @connection ||=
+        if @mailbox.imap?
```

Comment:
Can we swap these, so the check is for ms graphql, and the `else` is for imap? Super-minor thing.